Braintree.xcodeproj/project.pbxproj

```diff
@@ -5386,6 +5362,7 @@
 		A948D6E524FAC6C000F4F178 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
We shouldn't need this, [since Swift 5](https://www.swift.org/blog/abi-stability-and-apple/) (Xcode 10.2/ iOS 12.2) ships the Swift standard libraries with the OS, instead of needing it to be included with each app.

Reply:
Same thing here, this was generated automagically by Xcode. This line is for `BraintreeApplePayTests` in `Debug` and the other is for `Release`. It looks like this also matches all of our Swift tests. We should likely change this across the board in a follow up PR if we don't want this setting as it is.

Reply:
Got it - thanks for clarifying this is for the test targets